website/blog/2021-08-25-Using-the-Apache-APISIX-OpenID-Connect-Plugin-for-Centralized-Authentication.md

```diff
@@ -10,6 +10,7 @@ keywords:
 - Okta
 - 集中认证
 description: 本文展示了使用 Apache APISIX OpenID Connect Plugin 进行 Okta 集中认证的过程。 本文包含了对Apache APISIX 和 Okta 集中认证的概念介绍，展示了在 Apache APISIX 中直接建立来自 Okta 的集中式身份认证的过程。这种集中认证的方法减少了开发者的学习和维护成本，提供了安全和精简的用户体验。
+tags: [technology, practical case]
```

Comment:
Only in English?

Reply:
How about do the i18n of tags after the blogs i18n does?